external_services/lti.py

```diff
@@ -60,9 +62,9 @@ def __init__(self, service, user, instance, request, title, context_id=None, lin
             "launch_presentation_locale": get_language(),
             "launch_presentation_document_target":
                 "iframe" if exercise and exercise.open_in_iframe else "window",
-            "launch_presentation_return_url": request.scheme + '://' + request.get_host() + instance.get_absolute_url(),
+            "launch_presentation_return_url": settings.BASE_URL + instance.get_absolute_url(),
```

Comment:
You should consider using `urllib.parse.urljoin` for this or `parsed_url._replace(path=absolute_url)`. Both might yiedl more robust results (e.g. if there is or sin't slash in end of `BASE_URL`.